certora/specs/methods/IPausable.spec

```diff
@@ -0,0 +1,3 @@
+methods {
+    paused() returns (bool) envfree
+}
```

Comment:
I also think it's not needed, so I removed it and it's working as you suggested.

One question I have is, what are these files about if we're defining the interface inside each spec?

My feeling is that we should add `I*.spec` for standard interfaces (eg. `IERC20.spec` or `IERC5313.spec`) and import them where needed. 

An example would be: 

- I see `owner()` defined both in `IOwnable.spec` and `IOwnable2Step`, while they both can just import `IERC5313.spec`
- Pausable should not have an `IPausable.spec` because it doesn't adhere to any standard interface

Just a proposal, do you see this logic something we should follow more carefully?

Reply:
IMO the real question here is weither or not the interface is going to be reused. 

I'm not sure ERC5313 needs an interface here. On the other hand, having one for TimelockController makes sens if governor also needs it.

Reply:
Yes, I'm not suggesting adding the `IERC5313.spec`, but wanted to confirm what we expect before moving forward.

So far:

- If the interface is going to be reused, we create an `I*.spec`
- If it's not going to be reused, we put it at the top of its `*.spec` file